CHANGELOG.md

```diff
@@ -2,6 +2,32 @@
 
 ## Unreleased
 
+## v0.67.0
+
+This release changes the handling and availability of some command line parameters.  While we do not expect any negative customer impact or action required for existing applications to work, please cut us an issue on [github](https://github.com/signalfx/splunk-otel-collector/issues) if you experience any issues related to command line flags, ensuring to include a detailed error log and the context in which any failures happened..
+
+This Splunk OpenTelemetry Collector release includes changes from the [opentelemetry-collector v0.67.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.67.0) and the [opentelemetry-collector-contrib v0.67.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.67.0) releases where appropriate, in addition to various dependabot upgrades.
```

Comment:
```suggestion
This Splunk OpenTelemetry Collector release includes changes from the [opentelemetry-collector v0.67.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.67.0) and the [opentelemetry-collector-contrib v0.67.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.67.0) releases where appropriate.

### 🛑 Breaking changes 🛑
- Restrict Collector service command line arguments [#2343](https://github.com/signalfx/splunk-otel-collector/pull/2343) 
```

The dependabot updates have been included as

```
Third-party dependency updates
``

or similar

Reply:
+1 to this comment, we advertise breaking change with this type of header

Reply:
imo there is enough information in the item to signal advanced users who may be affected by the change.

for the github flow you could comment w/ a snippet or just push what you're proposing (since that's the aim of PRs) for the next comment cycle.

Reply:
I think our official support policy is opening a ticket, but we try to be reasonably responsive w/ gh issues: https://github.com/signalfx/splunk-otel-collector/blob/main/docs/troubleshooting.md

Reply:
No need for this contact/rfc disclaimers to be associated w/ this specific item imo.